app/controllers/foods_controller.rb

```diff
@@ -1,3 +1,84 @@
 class FoodsController < ApplicationController
-  def index; end
+  before_action :set_food, only: %i[show edit update destroy]
+
```

Comment:
Please refer to the requirements, the update operation should not be implemented
- A food list (with all CRUD implementation, except for 'update').